lib/_http_client.js

```diff
@@ -752,13 +739,10 @@ function requestOnPrefinish() {
 function emitFreeNT(req) {
   req._closed = true;
   req.emit('close');
-  if (req.res) {
-    req.res.emit('close');
-  }
-
   if (req.socket) {
     req.socket.emit('free');
   }
+
```

Comment:
Extraneous whitespace change should ideally be removed but I'm not going to block on that!

Reply:
Sorry, I'll fix it. Thanks.